### ISSUE TITLE
Crash by a Client Known with User-Agent Avira

### ISSUE THREAD

**Issue by kvic-z** [kvic-z/pixelserv-tls#8]
**Client**

Unsure but with User-Agent known as 'Avira'.

**Symptom**

pixelserv-tls v2.0.1-rc2 crashes when the client tries to POST contents.

**The request that caused crash**
```
'POST /engage/ HTTP/1.1
Host: api.mixpanel.com
User-Agent: Avira
Accept: */*
Content-Length:2130
Content-Type: application/x-www-form-urlencoded
Expect: 100-continue

'
socket:22 host:api.mixpanel.com
method: 'POST'
Segmentation fault
```

**Comment by kvic-z** [comment]
**Analysis**

No space between 'Content-Length:' and '2130'. I believe the standard requires a space in-between. Anyway pixelserv-tls shall have been more resilient in such case.

**Comment by kvic-z** [comment]
Fixed in [0063f766](https://github.com/kvic-z/pixelserv-tls/commit/0063f766c5ded3e56c536b325275be7cb48d3bd3)
